modules/control/submodules/postprocessor_submodule.cc

```diff
@@ -47,33 +47,18 @@ bool PostprocessorSubmodule::Init() {
 }
 
 bool PostprocessorSubmodule::Proc(
-    const std::shared_ptr<Preprocessor>& preprocessor_status,
     const std::shared_ptr<ControlCommand>& control_core_command) {
   ControlCommand control_command;
-  if (preprocessor_status->received_pad_msg()) {
-    control_command.mutable_pad_msg()->CopyFrom(
-        preprocessor_status->mutable_local_view()->pad_msg());
-  }
+  // get all fields from control_core_command for now
+  control_command.CopyFrom(*control_core_command);
 
-  // forward estop reason among following control frames.
-  // TODO(SJiang: remove preprocessor_status)
-  if (preprocessor_status->estop()) {
+  // estop handling
+  if (control_core_command->header().has_status()) {
```

Comment:
This is not right logic, you should only be checking whether a Estop is generated/passed from previous module